files/angular-pouchdb/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-pouchdb",
+  "repo": "angular-pouchdb/bower-angular-pouchdb",
+  "files": {
+    "include": ["angular-pouchdb.js", "angular-pouchdb.min.js"]
```

Comment:
Sorry I didnt notice that the repos are different. Then its fine, the bot will work.